pilot/pkg/networking/core/v1alpha3/extension/wasmplugin.go

```diff
@@ -52,6 +52,16 @@ func PopAppend(list []*hcm.HttpFilter,
 	return list
 }
 
+func BuildNetworkWasmFilters(filterMap map[extensions.PluginPhase][]*model.WasmPluginWrapper) []*listener.Filter {
+	filters := make([]*listener.Filter, 0)
+	for _, extensions := range filterMap {
```

Comment:
we should consider the phase  for tcp protocol

Reply:
yeah, It is in my list of things to do